### PR TITLE
Create first blog post on setup

### PR DESCRIPTION
**Pull request by juliaogris**
Rendered MarkDown at https://github.com/juliaogris/sysl/blob/blog-setup/docs/blog/setup.md

### CODE REVIEW

File: docs/blog/setup.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+The Sysl open source setup
+======================
+
+We have recently set up what we think of as good engineering practises for our enterprise-backed open source project [Sysl](https://github.com/anz-bank/sysl). The following is a record of systems and practises we consider essential and the products and platforms we have chosen (in parenthesis):
```
Comment (anzdaddy) [inline docs/blog/setup.md]: s/think of as/consider to be/

File: docs/blog/setup.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+* Issue tracking ([Github Issue](https://github.com/anz-bank/sysl/issues) tracking with [template](https://github.com/anz-bank/sysl/blob/master/ISSUE_TEMPLATE.md))
+* Project Managment ([Github projects](https://github.com/anz-bank/sysl/projects))
+* Documentation in same repo as sources ([README](https://github.com/anz-bank/sysl/blob/master/README.rst) and [docs/](https://github.com/anz-bank/sysl/blob/master/docs) as starting point)
+* Chat ([Gitter](https://gitter.im/anz-bank/sysl))
+* Status dashboard ([Badges](https://github.com/anz-bank/sysl/blob/master/README.rst))
+
+The most involved pieces have been automated quality ensurance and release automation. These parts of the Sysl project also keep evolving as we add new quality checks and artefact types. They deserve a closer look.
+
+Our branch model is simple - we develop feature branches on our own forks and issue pull requests to `master` on `anz-bank/sysl`. As suggested by [Github Flow](https://guides.github.com/introduction/flow/), "[t]here's only one rule: anything in the master branch is always deployable". Releases are linked to tags of commits in `master` on `anz-bank/sysl`.
```
Comment (anzdaddy) [inline docs/blog/setup.md]: *There's

File: docs/blog/setup.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+Our original (upsteram, parent) repository is owned by the Github orgaisation [`anz-bank`](https://github.com/anz-bank), which has got a `sysl-developer` team. Only team members have write access to `anz-bank/sysl` and can merge pull requests into `master`. Additionally, the `master` branch of  `anz-bank/sysl` is protected and has restrictions enabled to automate quality checks:
```
Comment (anzdaddy) [inline docs/blog/setup.md]: - *upstream
- *organization
- s/has got a/has a/

File: docs/blog/setup.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+ * Dismiss stale pull request approvals when new commits are pushed
+ * Require review from Code Owners
+ * Require branches to be up to date before merging
+ * Require status checks to pass before merging
+   - Require passing tests and no linting warning (on Travis CI Linux, Travis CI macOS, Appveyor Windows)
+   - Require stable or improved codecoverage
+
+Sysl releases are available on Sysl's [Github releases page](https://github.com/anz-bank/sysl/releases) and on various package registries (e.g. PyPI, BinTray, Docker Hub).  We follow [Semver](https://semver.org/) for versioning.
+
+Our release process has been automated with a script and artefact deployment by CI. A new release can be started with `src/scripts/release.sh prepare X.Y.Z` and after the automatically generated pull request is approved and merged	`src/scripts/release.sh deploy X.Y.Z` will create and push the release tag which will then trigger Travis and Appveyor to deploy the artefacts (see our [Releasing documentation](https://github.com/anz-bank/sysl/blob/master/docs/releasing.md) for more details).
```
Comment (anzdaddy) [inline docs/blog/setup.md]: - `s/with a script/with script/`
- `s/ and after/. After/`
- `s/merged\t/merged, /`
- `s/tag which/tag, which/`
- `s/our/the/`

File: docs/blog/setup.md (hunk 1 above)
Comment (juliaogris) [reply docs/blog/setup.md]: Done, but then we've got a word repetition in the first 2 sentences. So changed the second one to deem which os a strange word, isn't it?

File: docs/blog/setup.md (hunk 2 above)
Comment (juliaogris) [reply docs/blog/setup.md]: Done

File: docs/blog/setup.md (hunk 3 above)
Comment (juliaogris) [reply docs/blog/setup.md]: done, done, done

File: docs/blog/setup.md (hunk 4 above)
Comment (juliaogris) [reply docs/blog/setup.md]: done.

**Pull request merged**